packages/bodiless-organisms/src/components/Menu/SimpleMenu.tsx

```diff
@@ -57,30 +66,14 @@ const asMenuBase = (nodeKeys?: WithNodeKeyProps) => flow(
   asBodilessList(nodeKeys),
   asStylableList,
   withSubLists(1)({ SubMenu: asMenuSubList }),
-);
-
-/**
- * Applies the specified design to the main list and all sublists.
- *
- * @param design
- */
-const withMenuDesign = (design: Design<any>) => flow(
-  withSubListDesign(1)({
-    SubMenu: withDesign(design),
+  withMenuDesign({
+    Item: withoutProps(['addItem', 'deleteItem', 'canDelete']),
   }),
-  withDesign(design),
-);
-
-// Defines basic sub menu when displayed as main menu
-const asBasicSubMenu = flow(
-  asSubMenu,
-  withMenuItem,
 );
 
 // Applies above designs to the chameilion sublist
 const asChameleonSubMenu = withDesign({
-  SubMenu: asBasicSubMenu,
-  _default: asMenuItem,
+  SubMenu: asSubMenu,
```

Comment:
If we can get rid of asSubMenu, then we can probably get rid of the whole  `asMenuClean` (exported as `asSimpleMenu`/`asMegaMenu`) - and maybe use these name instead of `asSimpleMenuDefaultStyles`/`asMegaMenuDefaultStyles`..

Reply:
Done